.travis.yml.container

```diff
@@ -0,0 +1,27 @@
+language: python
+python:
+  - "2.7"
+
+addons:
+  apt:
+    packages:
+      - python-xapian
+      - python-apt
+      - python-cluster
+      - python-webpy
+      - python-simplejson
+      - python-numpy
+      - apt-xapian-index
+      - python-xdg debtags
+      - python-pip
+      - python-sklearn
+      - python-nltk
+      - python-matplotlib
+
+install: 
+  - update-apt-xanpian-index
+  - python -m nltk.downloader stopwords
```

Comment:
Since we no longer use nltk, I think both of these lines are not necessary anymore.